### ISSUE TITLE
Adding context to the aria-function attribute

### ISSUE THREAD

**Issue by ayelet-seeman** [ayelet-seeman/coga.personalisation#4]
We added the aria-function attribute, which enables the web developer to specify the element's function (e.g. home, first name, reset, etc.) 

However, we are concerned that developers will use aria-function values in incorrect contexts and they will be processed incorrectly as a result. 
For example, aria-function="contact us"  is intended to be treated as a link, but someone may add that to a heading with "contact us" causing it to be processed as if it were a link and mess the page.

We thought of two solutions:
A. Change the syntax of the aria-function attribute: following the name of the function have the context in which it should be used. For example, instead of having aria-function = "contact us" have aria-function = "contact us link".

B. Have the personalisation triggered only under the contexts we want to, by using tag names and roles.
For  example the aria-function="contact us" will trigger if in a link or if the role is either a link or navigation.
